scripts/issue/617/withdraw_all_rari.py

```diff
@@ -0,0 +1,37 @@
+from great_ape_safe import GreatApeSafe
+from helpers.addresses import registry
+from brownie import interface
+from rich.pretty import pprint
+
+
+safe = GreatApeSafe(registry.eth.badger_wallets.treasury_vault_multisig)
+
+
+def withdraw_usdc():
+    fusdc = interface.IFToken(registry.eth.rari['fUSDC-22'], owner=safe.account)
+
+    amount_fusdc = fusdc.balanceOf(safe)
```

Comment:
currently not all usdc is withdrawable, some is still being lent out:
<img width="456" alt="Screenshot 2022-07-13 at 12 10 12" src="https://user-images.githubusercontent.com/2835259/178709335-b47062db-d788-4e46-b6b4-534e0688c387.png">

not sure how the ui makes that calc and if we can make the same?

Reply:
yep looks like it will revert will trying to wd everything, [sim](https://dashboard.tenderly.co/petrovska/project/simulator/8e3e61d9-c62d-4544-b86b-ce7dbf831134/logs). [failure codes](https://docs.rari.capital/fuse/#failure-info)

i think it is doable by leveraging `getCash()` and its rate to ftoken to know how much is withdrawable at specific time

Reply:
fixed in [2527fc0](https://github.com/Badger-Finance/badger-multisig/pull/654/commits/2527fc0efface266c6fba6403bd89cd0f038b7ee), uses `getCash` along with `redeemUnderlying` instead